projects/memapCore/src/main/java/memap/components/prototypes/MEMAPCoordination.java

```diff
@@ -170,6 +186,69 @@ public AnswerContent returnAnswerContentToSend() {
 					e.printStackTrace();
 				}
 			}
+			
+			
+			// Option to save topology as json for usage in planning tool:
+			if (true) {			
```

Comment:
Do you still need this check? If not, I would remove the "if(true)" check, to reduce the nesting of if-s

Reply:
I was wondering if we introduce a flag later, wheather to save the topology or not, but i think we can agree to have it by default.

So yes, I will remove this if statement in the next commit. 👍